dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -240,8 +240,9 @@ public String buildInsertCommand(String tableName,
         return sb.toString();
     }
 
-    public DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors) {
-        return new DbStoredProcedureCall(this, name, accessors);
+    public StatementExecution newStoredProcedureCall(String name, DbParameterAccessor[] accessors)
+            throws SQLException {
+        return new DbStoredProcedure(this, name).buildPreparedStatement(accessors);
```

Comment:
`DbStoredProcedure` depends on the environment and the environment implementation here depends on `DbStoredProcedure`...

A more interesting point is that now there's nothing to call `newStoredProcedureCall`. If all works correctly in the current state (I'm not quite sure about Oracle since it's overriding this) - that method could rather be dropped.